auto-generated/cluster/stone-prd-rh01/tenants/rhtap-build-tenant/external-secrets.io_v1beta1_externalsecret_registry-pull-secret.yaml

```diff
@@ -0,0 +1,25 @@
+apiVersion: external-secrets.io/v1beta1
+kind: ExternalSecret
+metadata:
+  annotations:
+    argocd.argoproj.io/sync-options: SkipDryRunOnMissingResource=true
+    argocd.argoproj.io/sync-wave: "-1"
+  name: registry-pull-secret
+  namespace: rhtap-build-tenant
```

Comment:
Not sure it will work for this namespace. External Secrets are allowed only for a set of namespace. You have to change that setting of External Secrets operator too.

Reply:
Ah, I see, I'll take a look at the settings.

If it's not allowed, should we just manage the secret manually?

Reply:
It looks like our namespace is already there, if I'm looking at the right thing https://github.com/redhat-appstudio/infra-deployments/blob/8eafd619ea218208fea7135e698c0ec7eb2814a1/components/cluster-secret-store/base/appsre-stonesoup-vault-secret-store.yaml#L47